pkg/api/types.go

```diff
@@ -246,10 +246,27 @@ type OrchestratorVersionProfile struct {
 	OrchestratorProfile
 	// Whether this orchestrator version is deployed by default if orchestrator release is not specified
 	Default bool `json:"default,omitempty"`
+	// Whether version is in preview mode
+	IsPreview bool `json:"isPreview,omitempty"`
+	// Metadata for customers to help them select a healthy kubernetes version
```

Comment:
this comment seems a bit vague, what does "help them select a healthy kubernetes version" mean?

what is a "healthy" k8s version? 

Kubernetes should start with an upper case.

Reply:
Yes, it is vague. Jake's team has asked me for this parameter to allow them to provide customers with additional information. When you select a kubernetes version, you also happen to selecting the versions for many other components that are bundled. He wants to be able to inform customers of problems with any of those versions via this metadata. That way customers can make informed decisions about the version they select. 

I will upper case it and ask the boulder team if there is a better comment that could be used instead.